msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -45,26 +44,65 @@ Users must not enter a parameter for a header or cookie. If you need to pass hea
 
 <details><summary>2. App manifest</summary>
 
+* Set the app manifest version to `devPreview`.
 * Set `composeExtensions.composeExtensionType` to `apiBased`.
 * Define `composeExtensions.apiSpecificationFile` as the relative path to the OpenAPI Description file within the folder.
+* Define `apiSpecificationFile` as the relative path to the OpenAPI description document.
 * Define `apiResponseRenderingTemplateFile` as the relative path to the response rendering template.
 * Each command must have a link to the response rendering template.
 * Full description must not exceed 128 characters.
 * A command must have exactly one parameter.
+* Add a new optional property `authorization` under the `composeExtensions` node. This property should be null for bot-based ME and specified only for API ME.
+* Define the type of authentication your application uses by setting the `authType` property under the `authorization` node. The possible values for this property are `none`, `apiSecretServiceAuth`, `microsoftEntra`, and `userAuth`.
+* Depending on the type of authentication your application uses, you might need to add a corresponding configuration object under the `authorization` node. For example, if your application uses AAD SSO, you would add a `microsoftEntraConfiguration` object with a `supportsSingleSignOn` property set to `true`.
+* If your application uses API key based authentication, you would add an `apiSecretServiceAuthConfiguration` object with an `apiSecretRegistrationId` property. This property should contain the reference ID returned when you submitted the API key through the portal.
 
 </details>
 
 </br>
 
 <details><summary>3. Response rendering template</summary>
 
+> [!NOTE]
+> Teams supports Adaptive Cards up to version 1.5, and  the Adaptive Cards Designer supports up to version 1.6.
+
+* Use tools such as Fiddler or Postman to call the API and ensure that the request and the response are valid.
```

Comment:
What is the request/response refer to? Are they sent to Teams client or the adaptive card designer? I'm not sure whether the developer can test this without providing a valid token.